scripts/utils/board_v1_to_v2.py

```diff
@@ -157,11 +158,15 @@ def board_v1_to_v2(board_root, board, new_board, group, vendor, soc, variants):
 
     print(f"Creating or updating Kconfig.{new_board}...")
     new_board_kconfig_file = new_board_path / f"Kconfig.{new_board}"
-    selects = "\n\t" + "\n\t".join(["select " + setting for setting in board_soc_settings])
+    header = "# SPDX-License-Identifier: Apache-2.0\n"
```

Comment:
I wonder if we should just re-use copyright from the deleted "Kconfig.board"

Reply:
You're right. I'll have another shot.

Reply:
@gmarull What if we just:
- move and rename the existing Kconfig.board
- drop the "bool"
- s/depends on/select

Reply:
some selects will need to be removed (e.g. all SOC_SERIES*), I need to update the script for that. So not sure if easier to update the file or regenerate it.

Reply:
On some boards, we'd need to remove the help as well indeed. See https://github.com/zephyrproject-rtos/zephyr/pull/67737/files

Reply:
so then I'd just grab the copyright and create a new file as we do now

Reply:
> some selects will need to be removed (e.g. all SOC_SERIES*), I need to update the script for that. So not sure if easier to update the file or regenerate it.

Do you think about something like follows ?
```
-            m = re.match(r"^CONFIG_(SOC_[A-Z0-9_]+).*$", line)
+            m = re.match(r"^CONFIG_(SOC_(?!SERIES_)[A-Z0-9_]+).*$", line)
```

Reply:
yes, willing to add a patch :-)?

Reply:
Done ;)